pytest_girder/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+setup(
+    name='pytest-girder',
+    description='A set of pytest fixtures for testing Girder applications.',
+    author='Kitware, Inc.',
+    author_email='kitware@kitware.com',
+    license='Apache 2.0',
+    classifiers=[
+        'Framework :: Pytest'
+    ],
+    install_requires=[
+        'girder',
+        'pytest'
+    ],
+    entry_points={
+        'pytest11': [
+            'girder = pytest_girder.plugin'
+        ]
+    })
```

Comment:
We'll almost certainly need to push this to pypi for external plugins to use.  It can wait until we work out some of the plugin refactoring, but we need to figure out a release workflow... Are the versions synchronized with girder versions?  If not, when do we push these releases?